content/en/blog/contributor-contest/index.md

```diff
@@ -0,0 +1,133 @@
++++
+title = "KubeEdge contributor contest"
+subtitle = "To encourage open source community members participation in KubeEdge project, this contest is been conducted. KubeEdge maintainers welcome community members, individuals, organizations & students for this contest. Each winner will get a free pass to [KubeCon + CloudNativeCon & Open source summit, Shanghai, China 2019.](https://www.lfasiallc.com/events/kubecon-cloudnativecon-china-2019/) The contest will start on **20th April 2019** and end on **23rd May 2019.**"
+date = 2019-04-20
+lastmod = 2019-04-20
+
+draft = false
+
+# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
+authors = ["KubeEdge"]
+
+# Tags and categories
+# For example, use `tags = []` for no tags, or the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
+tags = ["KubeEdge", "kubeedge", "edge computing", "kubernetes edge computing", "K8S edge orchestration", "edge computing platform", "KubeCon", "cloud native"]
+categories = ["General", "Announcements"]
+summary = "To encourage open source community members participation in KubeEdge project, this contest is been conducted. KubeEdge maintainers welcome community members, individuals, organizations & students for this contest. The event will start on 20th April 2019 and end on 23rd May 2019"
```

Comment:
need to mention time as well along with date. Like in IST or GMT orelse there can be confusion.

Reply:
Added.